openforcefield/topology/molecule.py

```diff
@@ -739,8 +845,8 @@ def __repr__(self):
 
     def __str__(self):
         # TODO: Also include particle_index, which molecule this atom belongs to?
-        return "<VirtualSite name={} type={} atoms={}>".format(
-            self.name, self.type, self.atoms
+        return "<VirtualSite name={} type={} atoms={} particles={}>".format(
+            self.name, self.type, self.atoms, self.n_particles
```

Comment:
Virtually identical to `__repr__`, except out of sync now. I have little preference on which to use, but they should be consistent

```python3
In [18]: VirtualSite(atoms=[mol.atoms[0]])
Out[18]: VirtualSite(name=None, type=VirtualSite, atoms=[Atom(name=, atomic number=8)])

In [19]: str(VirtualSite(atoms=[mol.atoms[0]]))
Out[19]: '<VirtualSite name=None type=VirtualSite atoms=[Atom(name=, atomic number=8)] particles=1>'

```